import_export_extensions/widgets.py

```diff
@@ -325,3 +324,9 @@ def _get_file(self, url: str) -> File:
         filename = f"{self.filename}.{ext}" if ext else self.filename
 
         return File(file, filename)
+
+    def _get_default_storage(self, settings: LazySettings) -> str:
```

Comment:
Does it make sense to pass settings as argument?

Reply:
Hmm, I tried to avoid implicit dependency of global variable. But actually it won't help, I'll remove it